src/components/Chat.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { FaSearch } from 'react-icons/fa';
+
+import DialogItem, { DialogItemProps } from 'components/DialogItem';
+import { Divider, FlexRow } from 'components/CommonStyledComponents';
+
+const ChatContainer = styled.div`
+  padding-top: 75px;
+  padding-left: 75px;
+  padding-right: 10px;
```

Comment:
```suggestion
  padding: 75px 10px 0 75px;
```

if you have three rules you might as well use a one liner